### PR TITLE
feature/sd-1672 - Updated ResourceNamespaceInfo.json

### PR DESCRIPTION
**Pull request by packages-simeon**

### CODE REVIEW

File: ResourceNamespaceInfo.json [hunk 1]
```diff
@@ -1019,7 +1019,7 @@
     "namespace": "PowerShell:SecurityAndComplianceCenter:RetentionCompliancePolicies",
     "description": "Office 365 > Security & Compliance > Information governance > Retention",
     "url": "https://compliance.microsoft.com/informationgovernance?viewid=retention",
-    "isCreateSupportedFilter": ".*",
+    "isCreateSupportedFilter": "$^",
```
Comment (jeff-simeon) [inline ResourceNamespaceInfo.json:1022]: Why this change?

**Review by rdorta-simeon — APPROVED** [review]

File: ResourceNamespaceInfo.json (hunk 1 above)
Comment (rdorta-simeon) [reply ResourceNamespaceInfo.json:1022]: new cmdlets on SCC for these dont work

**Review by jeff-simeon — APPROVED** [review]
